frontend/packages/console-app/src/components/user-preferences/language/LanguageDropdown.scss

```
@@ -0,0 +1,3 @@
+.co-user-preference-language-checkbox {
```

This should follow BEM naming, something like

```suggestion
.co-language-dropdown__system-default-checkbox {
```

http://getbem.com/naming/